code/datums/status_effects/drug_effects.dm

```diff
@@ -58,3 +58,39 @@
 	name = "Seizure"
 	desc = "FJOIWEHUWQEFGYUWDGHUIWHUIDWEHUIFDUWGYSXQHUIODSDBNJKVBNKDML <--- this is you right now"
 	icon_state = "paralysis"
+
+/datum/status_effect/stoned
+	id = "stoned"
+	duration = 10 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/stoned
+	status_type = STATUS_EFFECT_REFRESH
+	var/original_eye_color
+
+/datum/status_effect/stoned/on_apply()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		original_eye_color = human_owner.eye_color
+		human_owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/cannabis) //slows you down
+		human_owner.eye_color = BLOODCULT_EYE //gives cultists plausible deniability
```

Comment:
Do they really have plausible deniability if the bloodshot trait is added?

Reply:
Trying to recall my rationale. No, there's no plausible deniability unless the person examining them doesn't read the red eyes examine message. It makes a bit more ambiguous, though.

Reply:
Maybe change it so there is, or alter the comment, then.